### DISCUSSION TITLE
Help wanted - How to install on All-Inkl Shared Hosting?

### DISCUSSION THREAD

**Discussion by SunboX** [shlinkio/shlink#2144] [Help wanted] [answered]
### Shlink version

4.1.1

### PHP version

8.3

### How do you serve Shlink

Other (explain in summary)

### Database engine

MariaDB

### Database version

10.6.18

### Summary

Hi,

I'm using https://all-inkl.com/ shared webhosting. It supports PHP and I have a MySQL database. But I can not access a command line / shell and also I can not set env variables.

How can I install / setup shlink?

greetings and thanks,
André

**Comment by acelaya** [comment, marked as answer]
With those limitations, I'm afraid most probably you can't.

There are URL shorteners better suited for that kind of environment, like YOURLS.
